Runtime/Behaviours/CinemachineDollyCart.cs

```diff
@@ -20,7 +22,10 @@ public class CinemachineDollyCart : MonoBehaviour
     {
         /// <summary>The path to follow</summary>
         [Tooltip("The path to follow")]
-        public CinemachinePathBase m_Path;
+        public SplineContainer m_Path;
+        //public CinemachinePathBase m_Path;
```

Comment:
you can't leave in commented-out lines.  I assume that if this were a real PR, these wouldn't be here